components/item/MetadataField.vue

```diff
@@ -33,13 +33,29 @@
             />
           </li>
         </template>
+        <template v-else-if="typeof fieldData.url !== 'undefined'">
+          <li
+            :key="index"
+            :lang="langMappedValues.code"
+            data-qa="literal value"
+          >
+            <SmartLink
+              :destination="fieldData.url"
+              :link-class="name === 'edmDataProvider' ? 'view-at' : null"
```

Comment:
Should this link show an icon as it is/could be an external link?

Reply:
ok I've added the icon